scripts/update-version.sh

```diff
@@ -9,26 +9,17 @@
 set -e 
 
 NEW_VERSION="$1"
```

Comment:
can we possibly add these checks to point out what issue can occur? 

```
if [ -z "$1" ]
then
  echo "Error: No version number supplied"
  echo "Usage: $0 [new version number]"
  exit 1
fi

if ! command -v sd &> /dev/null
then
    echo "sd command not found. Install it using 'brew install sd'"
    exit 1
fi
```

Reply:
If the script was going to be used by developers on their development machines, I could see value in adding code for error checking. 

Because this is a script only used by the CI, IMO, adding error checking code is over engineering and adds complexity to the bash script. Instead, documentation in the script provides what the developer needs to do for the CI to run the script successfully.